<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AliasesWizardPage</class>
 <widget class="QWizardPage" name="AliasesWizardPage">
  <property name="title">
   <string>Command aliases</string>
  </property>
  <property name="subTitle">
   <string>Add and remove convenient command aliases</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolButton" name="toolButtonAdd">
     <property name="icon">
      <iconset resource="../../communi.qrc">
       <normaloff>:/resources/oxygen/16x16/actions/add.png</normaloff>:/resources/oxygen/16x16/actions/add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="toolButtonEdit">
     <property name="icon">
      <iconset resource="../../communi.qrc">
       <normaloff>:/resources/oxygen/16x16/actions/edit.png</normaloff>:/resources/oxygen/16x16/actions/edit.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButtonRemove">
     <property name="icon">
      <iconset resource="../../communi.qrc">
       <normaloff>:/resources/oxygen/16x16/actions/remove.png</normaloff>:/resources/oxygen/16x16/actions/remove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Filter:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditFilter</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="lineEditFilter"/>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QTreeView" name="treeView">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../communi.qrc"/>
 </resources>
 <connections/>
</ui>
